src/radio.cpp

```diff
@@ -100,18 +100,18 @@ void zmq::radio_t::xwrite_activated (pipe_t *pipe_)
 
 void zmq::radio_t::xpipe_terminated (pipe_t *pipe_)
 {
-    for (subscriptions_t::iterator it = subscriptions.begin (); it != subscriptions.end (); ) {
```

Comment:
OK, this project is fancy due to lot of dependencies from legacy compilers. But still I will try to contribute some code to it